### PR TITLE
Add a section for deprecated packages in README

### PR DESCRIPTION
**Pull request by bbrto21**
* Change all plugins of fluttercommunity from 3rd-party to 1st-party.

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,30 +37,37 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
...
+| [**device_info_tizen**](packages/device_info) | [![pub package](https://img.shields.io/pub/v/device_info_tizen.svg)](https://pub.dev/packages/device_info_tizen) | [**device_info_plus_tizen**](packages/device_info_plus) | [![pub package](https://img.shields.io/pub/v/device_info_plus_tizen.svg)](https://pub.dev/packages/device_info_plus_tizen) |
+| [**package_info_tizen**](packages/package_info) | [![pub package](https://img.shields.io/pub/v/package_info_tizen.svg)](https://pub.dev/packages/package_info_tizen) | [**package_info_plus_tizen**](packages/package_info_plus) | [![pub package](https://img.shields.io/pub/v/package_info_plus_tizen.svg)](https://pub.dev/packages/package_info_plus_tizen) |
+| [**sensors_tizen**](packages/sensors) | [![pub package](https://img.shields.io/pub/v/sensors_plus_tizen.svg)](https://pub.dev/packages/sensors_plus_tizen) | [**sensors_plus_tizen**](packages/sensors_plus) | [![pub package](https://img.shields.io/pub/v/share_plus_tizen.svg)](https://pub.dev/packages/share_plus_tizen) |
```
Comment (swift-kim) [inline README.md:50]:
```suggestion
| [**sensors_tizen**](packages/sensors) | [![pub package](https://img.shields.io/pub/v/sensors_tizen.svg)](https://pub.dev/packages/sensors_tizen) | [**sensors_plus_tizen**](packages/sensors_plus) | [![pub package](https://img.shields.io/pub/v/sensors_plus_tizen.svg)](https://pub.dev/packages/sensors_plus_tizen) |
```

**Review by swift-kim — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,26 +15,20 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
...
-| [**package_info_tizen**](packages/package_info) | [package_info](https://github.com/flutter/plugins/tree/master/packages/package_info) (1st-party) | [![pub package](https://img.shields.io/pub/v/package_info_tizen.svg)](https://pub.dev/packages/package_info_tizen) | No |
-| [**package_info_plus_tizen**](packages/package_info_plus) | [package_info_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/package_info_plus) (3rd-party) | [![pub package](https://img.shields.io/pub/v/package_info_plus_tizen.svg)](https://pub.dev/packages/package_info_plus_tizen) | No |
+| [**network_info_plus_tizen**](packages/network_info_plus) | [network_info_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/network_info_plus) (1st-party) | [![pub package](https://img.shields.io/pub/v/network_info_plus_tizen.svg)](https://pub.dev/packages/network_info_plus_tizen) | No |
```
Comment (HakkyuKim) [inline README.md:26]: `network_info_plus` replaces `wifi_info_flutter`.
https://github.com/flutter/plugins/tree/master/packages/wifi_info_flutter/wifi_info_flutter

Comment (bbrto21) [reply README.md:26]: I Added row for wifi_info_flutter to deprecated table.

Comment (HakkyuKim) [reply README.md:26]: Thanks, you should also remove it from "Device Limitations".

Comment (bbrto21) [reply README.md:26]: Thank you, I removed it

**Pull request merged**